shared/models/qr_code.yml

```
@@ -39,3 +39,26 @@ properties:
   pages:
     type: string
     description: Specify the pages where the QR code should be stamped in a comma separated format. Your QR code can be printed in the same position on multiple pages. For postcards, the values should either be "front", "back" (for either front or back) or "front,back" (for the QR code to be printed on both sides). For self-mailers, the values should either be "inside", "outside" (for either inside or outside) or "inside,outside" (for the QR code to be printed on both sides). For letters, the values can be specific page numbers ("1", "3"), page number ranges such as "1-3", or a comma separated combination of both ("1,3,5-7").
+
+  logo:
+    type: object
+    description: Allows specifying a logo to be placed over top of the center of the QR code.  Logos must be in PNG or JPEG format and have a maximum file size of 1 MB.
```

would it be "over the center" or "on top of the center"?

```suggestion
    description: Allows specifying a logo to be placed over the center of the QR code.  Logos must be in PNG or JPEG format and have a maximum file size of 1 MB.
```